harmonica/_forward/tesseroid.py

```diff
@@ -51,21 +50,15 @@ def tesseroid_gravity(
         It is equivalent to the opposite of the radial component, therefore
         it's positive if the acceleration vector points inside the spheroid.
 
-    .. important::
-
-        - The gravitational potential is returned in
-          :math:`\text{J}/\text{kg}`.
-        - The gravity acceleration components are returned in mgal
-          (:math:`\text{m}/\text{s}^2`).
-
     Parameters
     ----------
-    coordinates : list of arrays
-        List of arrays containing the ``longitude``, ``latitude`` and
-        ``radius`` coordinates of the computation points, defined on
-        a spherical geocentric coordinate system. Both ``longitude`` and
-        ``latitude`` should be in degrees and ``radius`` in meters.
-    tesseroids : list or 2d-array
+    coordinates : list or 1d-array
+        List or array containing ``longitude``, ``latitude`` and ``radius`` of
+        the computation points defined on a spherical geocentric coordinate
+        system.
+        Both ``longitude`` and ``latitude`` should be in degrees and ``radius``
+        in meters.
+    tesseroids : list or 1d-array
```

Comment:
I think there were some revert changes from one of the last PRs in `main`. I'm suggesting to apply them back.
```suggestion
    coordinates : list of arrays
        List of arrays containing the ``longitude``, ``latitude`` and
        ``radius`` coordinates of the computation points, defined on
        a spherical geocentric coordinate system. Both ``longitude`` and
        ``latitude`` should be in degrees and ``radius`` in meters.
    tesseroids : list or 2d-array
```